<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Vyro-MacBook.local" failures="1" tests="8" name="Testcases.px_bg_object_mainscreen" time="121.814" errors="0" timestamp="2021-09-01T10:32:39 PKT" skipped="0">
  <testcase classname="Testcases.px_bg_object_mainscreen" name="A_Testcase1" time="4.162"/>
  <system-out/>
  <testcase classname="Testcases.px_bg_object_mainscreen" name="B_Testcase2" time="10.155"/>
  <system-out/>
  <testcase classname="Testcases.px_bg_object_mainscreen" name="F_Testcase6" time="39.760"/>
  <system-out/>
  <testcase classname="Testcases.px_bg_object_mainscreen" name="H_Testcase8" time="9.434"/>
  <system-out/>
  <testcase classname="Testcases.px_bg_object_mainscreen" name="G_Testcase7" time="12.780"/>
  <system-out/>
  <testcase classname="Testcases.px_bg_object_mainscreen" name="E_Testcase5" time="3.864"/>
  <system-out/>
  <testcase classname="Testcases.px_bg_object_mainscreen" name="D_Testcase4" time="7.958">
    <failure message="" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at org.testng.AssertJUnit.fail(AssertJUnit.java:65)
at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:23)
at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)
at Testcases.px_bg_object_mainscreen.D_Testcase4(px_bg_object_mainscreen.java:149)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- D_Testcase4 -->
  <system-out/>
  <testcase classname="Testcases.px_bg_object_mainscreen" name="C_Testcase3" time="33.701"/>
  <system-out/>
</testsuite> <!-- Testcases.px_bg_object_mainscreen -->
